test/Microsoft.AspNet.Mvc.TagHelpers.Test/ScriptTagHelperTest.cs

```diff
@@ -288,8 +326,8 @@ public static TheoryData RunsWhenRequiredAttributesArePresent_Data
             await helper.ProcessAsync(context, output);
 
             // Assert
-            Assert.Null(output.TagName);
```

Comment:
As in the `<link>` tag helper tests, please split the cases up enough that `output.TagName` can be tested.

One option would be to pass in the expected tag name i.e. add it to the `TheoryData`. That might make things easier.